README.md

```diff
@@ -21,20 +30,20 @@ To call them, simply type the name of the script without the `.py` extension.
 
 The two main scripts used in the paper are:
```

Comment:
What about the other scripts? Were they considered extraneous for the preprint?

Reply:
If I recall correctly, I think some of these scripts were actually last minute analyses that my Thesis Committee wanted to see for my defense.

I might just remove these and put them in another repo.

Reply:
It might be enough to just put them in a new branch.

Reply:
Unnecessary files are removed and additional comments added for other script files in https://github.com/tsalo/medic_analysis/pull/1